pegasus/sites.v3/code.org/public/teacher-dashboard/index.haml

```diff
@@ -102,7 +102,10 @@ angular: teacherDashboard
       //  'remove': {method:'DELETE'},
       //  'delete': {method:'DELETE'}
          update: {method:'POST', url: 'v2/sections/:id/update'},
+         // TODO: This isn't very restful
+         allStudents: {method:'GET', url:'v2/sections/:id/students', isArray: true},
          addStudents: {method:'POST', url:'/v2/sections/:id/students', isArray: true},
+         moveStudents: {method:'POST', url:'/sections/:new_section_code/transfers'},
```

Comment:
The existing v2/sections api is on code.org/localhost.code.org and your new api is on studio.code.org/localhost.studio.code.org.
Try something like:
`url: CDO.studio.url("sections/:new_section_code/transfers")`